docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -2,27 +2,36 @@
+    
+!!! tip
+
+    Work is being done to simplify this further to not have to upgrade via 1.7LTS, eta on that is early fall 2019. However all the tools needed to bring you from 4.x and 5.x to Platform, with or withouth legacy bridge, already exists so you can safly consider it ready for use already.
 
 !!! caution "Things to be aware of when planning a migration"
 
     1. While the instructions below are fully supported, we are aware that the community, partners and customers come from a wide range of different versions of eZ Publish, some with issues that do not surface before attempting a migration. That's why we and the community are actively gathering feedback on Slack and/or support channels for Enterprise customers to gradually improve the migration scripts and instructions. Reach out before you start so others who have done this before you can support you.
 
-    1. As of eZ Platform v1.11, Legacy Bridge is a supported option for 1.x and future 2.x series. This means you can plan for a more gradual migration if you want, just like you could on eZ Publish Platform 5.x, with a more feature-rich version of eZ Platform and (with 2.x) also more recent version of Symfony. This is a great option for those who want the latest features and are comfortable with more frequent releases.
+    2. "Hybrid setup" using Legacy Bridge is a supported option for 1.13LTS and 2.5LTS series. This means you can plan for a more gradual migration if you want, just like you could on eZ Publish Platform 5.x, with a more feature-mature version of eZ Platform, Symfony and even Legacy bridge itself. This is a great option for those who want the latest features. The downside is a more complex setup to develop and maintian, given you run two systems in paralell, and the overall migration might take much longer doing using an iterative/gradual approach. 
 
-    1. Additionally there are some other topics to be aware of for the code migration from eZ Publish to eZ Platform:
+    3. Additionally there are some other topics to be aware of for the code migration from eZ Publish to eZ Platform:
 
         - [Field Types reference](../api/field_type_reference.md) for overview of Field Types that do and don't exist in eZ Platform
-        - eZ Platform RichText Field Type capabilities, currently not covering [Custom Tags](https://jira.ez.no/browse/EZP-25357)
-        - Symfony 2.8, this is also the case on later 5.4.x versions, but not the first ones including 2014.11
-        - API changes. While we have a strict backwards compatibility focus, some deprecated API features were removed and some changes were done to internal parts of the system. See [ezpublish-kernel:doc/bc/changes-6.0.md](https://github.com/ezsystems/ezpublish-kernel/blob/v6.7.0/doc/bc/changes-6.0.md)
+        - eZ Platform RichText Field Type capabilities: You'll need v2.5.1 or higher for custom attributes migration support.
```

Comment:
I think it's good to mention that you need 2.5.1 if you want custom attributes. But if you do `composer update` you will automatically have the latest release, right?
I don't see why people will stay on 2.5.0 during the migration.

Reply:
You are right, that comment is not longer needed (it was written at the time 2.5.1 came out).

Reply:
```suggestion
        - eZ Platform RichText Field Type capabilities: You'll need latests v2.5LTS for custom attributes migration support.
```